src/ssl.c

```diff
@@ -2721,6 +2721,12 @@ void wolfSSL_ERR_print_errors_fp(FILE* fp, int err)
     fprintf(fp, "%s", data);
 }
 
+#if defined(OPENSSL_EXTRA) || defined(DEBUG_WOLFSSL_VERBOSE)
+void wolfSSL_ERR_dump_errors_fp(FILE* fp)
```

Comment:
wc_last_error_line and wc_last_error_file are accessed in this file (ssl.c) without mutex protection.  This is dangerous.